src/test/K6/src/api/storage/applications.js

```diff
@@ -3,46 +3,53 @@ import * as config from "../../config.js";
 import * as header from "../../buildrequestheaders.js"
 
 //Api call to Storage:Applications for an Org and returns response
-export function getAllApplications(altinnStudioRuntimeCookie, appOwner){
+export function getAllApplications(altinnStudioRuntimeCookie, appOwner) {
     var endpoint = config.platformStorage["applications"] + "/" + appOwner;
-    var params = header.buildHearderWithRuntime(altinnStudioRuntimeCookie, "platform");    
+    var params = header.buildHearderWithRuntime(altinnStudioRuntimeCookie, "platform");
     return http.get(endpoint, params);
 };
 
 //Api call to Storage:Applications to get app metadata by appName and returns response
-export function getAppByName(altinnStudioRuntimeCookie, appOwner, appName){
+export function getAppByName(altinnStudioRuntimeCookie, appOwner, appName) {
     var endpoint = config.platformStorage["applications"] + "/" + appOwner + "/" + appName;
-    var params = header.buildHearderWithRuntime(altinnStudioRuntimeCookie, "platform");    
+    var params = header.buildHearderWithRuntime(altinnStudioRuntimeCookie, "platform");
     return http.get(endpoint, params);
 };
 
 //Api call to Storage:Applications to create an application with app metadata and returns response code
-export function postCreateApp(altinnStudioRuntimeCookie, appOwner, appName, metadata){
+export function postCreateApp(altinnStudioRuntimeCookie, appOwner, appName, metadata) {
     var endpoint = config.platformStorage["applications"] + "?appId=" + appOwner + "/" + appName;
     var params = header.buildHearderWithRuntimeandJson(altinnStudioRuntimeCookie, "platform");
     var requestBody = JSON.stringify(metadata);
     return http.post(endpoint, requestBody, params);
 };
 
 //Api call to Storage:Applications to Edit an application with app metadata and returns response code
-export function putEditApp(altinnStudioRuntimeCookie, appOwner, appName, metadata){
+export function putEditApp(altinnStudioRuntimeCookie, appOwner, appName, metadata) {
     var endpoint = config.platformStorage["applications"] + "/" + appOwner + "/" + appName;
     var params = header.buildHearderWithRuntimeandJson(altinnStudioRuntimeCookie, "platform");
     var requestBody = JSON.stringify(metadata);
     return http.put(endpoint, requestBody, params);
 };
 
+//Api call to Storage:Applications to delete an app by appName and returns response
+export function deleteAppByName(altinnStudioRuntimeCookie, appOwner, appName, hardDelete) {
```

Comment:
hmm this endpoint should only be available through the designer.. Have you had successfull tests authenticated as an app owner?

Reply:
yes, we have an exposed endpoint in storage to delete an app. But the test is a negative test that validates for 403 response code.